miner/currency_blocklist.go

```diff
@@ -39,7 +39,7 @@ func (b *AddressBlocklist) FilterAllowlist(allowlist common.AddressSet, latest *
 			filtered[a] = struct{}{}
 		}
 	}
-	return filtered
+	return filtered, true
```

Comment:
What's the purpose of this? It seems like an unfinished thought

Reply:
Yes, true it's not doing anything in that PR - but this stems from the "enable" blocklist feature we pulled out.
I think it is helpful in that case to return not only the filtered allowlist, but also wether the "enable" feature was true or not during filtering and during the same lock acquisition.

Maybe at this point we should store the enable flag somewhere else, e.g. in the miner and guard the `FilterAllowlist` method with an if condition depending on that flag.